Hash_map/include/CGAL/Hash_map/internal/chained_map.h

```diff
@@ -37,42 +37,33 @@ class chained_map_elem
 template <typename T, typename Allocator>
 class chained_map
 {
-   const std::size_t nullptrKEY;
-   const std::size_t NONnullptrKEY;
-
-   chained_map_elem<T> STOP;
+   static constexpr std::size_t nullkey = (std::numeric_limits<std::size_t>::max)();
```

Comment:
I get some warning about this line:
```
chained_map.h:40:49: error: implicit instantiation of undefined template 'std::numeric_limits<unsigned long>'
max_size_type.h:53:10: note: template is declared here
```
If I add `#include <limits>` I then get:
```
chained_map.h:41:33: error: constexpr variable 'nullkey' must be initialized by a constant expression
limits:1274:57: note: value 18446744073709551614 is outside the range of representable values of type 'long long'
chained_map.h:41:43: note: in call to 'max()'
```

Wasn't sure how to fix it but 
```c++
static constexpr std::size_t nullkey = SIZE_MAX;
```
Seems to work.